crates/tests/test-tedge-cli/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "test-tedge-cli"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rexpect = { git = "https://github.com/didier-wenzek/rexpect", branch = "fix/tokenization" }
```

Comment:
Unfortunately, [rexpect](https://lib.rs/crates/rexpect) has some bugs and seems no more maintained.

Reply:
This has been fixed. Nice!